### ISSUE TITLE
Rofi configuration error, syntax error in the theme

### ISSUE THREAD

**Issue by galou** [justahuman1/i3-grid#6]
When launching `manager.sh`, I get a parsing error with `matrix.rasi`: syntax error line 50 column 6 to column 8 (`bg: ${BG};`). I changed to `bg: rgba(0,0,0,100%);` but the error was then on next line.

I have rofi 1.5.0.

Thanks for providing i3-grid by the way!

**Comment by justahuman1** [comment]
Hi,

Please see the following response to this [error](https://github.com/justahuman1/i3-grid/issues/2#issuecomment-644537325). 

TLDR; You will need rofi v1.5.2. You can stay with v1.5.0 if you replace all the environment variables with static CSS attributes (such as you have done with replacing line 50). A [pastebin](https://pastebin.com/raw/QAZ7pJSz) with all the variables replaced (for the default UI as seen in the README) if you do not want to update Rofi.

Additional Information: The [dev](https://github.com/justahuman1/i3-grid/commit/4b63614635e32357771fdfb5b3b003094fd6c746) branch (currently unstable) is utilizing the environment variables for dynamic menu shapes for the next version (0.2.4b1). I would suggest updating rofi (it's really useful for dynamic UI's) but
it's not 100% necessary for the core UI functionality.

Hope this helps! Feel free to also post any requests for future updates of i3-grid.
